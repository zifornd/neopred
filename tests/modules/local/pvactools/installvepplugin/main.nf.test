nextflow_process {

    name "Test Process PVACTOOLS_INSTALLVEPPLUGIN"
    script "modules/local/pvactools/installvepplugin/main.nf"
    process "PVACTOOLS_INSTALLVEPPLUGIN"

    test("Should run PVACTOOLS_INSTALLVEPPLUGIN without failures") {

        when {
            process {
                """
                //input[0] = file("$projectDir/tests/test_result/local/quants/test_input_file.pm")
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert process.out.results },
                { assert process.out.versions },
		{assert snapshot(process.out).match() }
            )
        }

    }

}
