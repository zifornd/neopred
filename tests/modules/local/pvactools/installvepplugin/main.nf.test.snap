{
    "Should run PVACTOOLS_INSTALLVEPPLUGIN without failures": {
        "content": [
            {
                "0": [
                    [
                        "Frameshift.pm:md5,7a8bd8e48f2b39a83232bac5400cf10f",
                        "Wildtype.pm:md5,685ad5086e3f96ec92127dec78e4aecb"
                    ]
                ],
                "1": [
                    "versions.yml:md5,5c78951211e41c97da0a18dd8feb20b5"
                ],
                "results": [
                    [
                        "Frameshift.pm:md5,7a8bd8e48f2b39a83232bac5400cf10f",
                        "Wildtype.pm:md5,685ad5086e3f96ec92127dec78e4aecb"
                    ]
                ],
                "versions": [
                    "versions.yml:md5,5c78951211e41c97da0a18dd8feb20b5"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2025-01-06T07:13:12.43427"
    }
}